chapters/planning-data-work.tex

```diff
@@ -1,147 +1,457 @@
-
-% When using a computer for research, you should keep in mind a structure of work known as \textbf{scientific computing}.\cite{wilson2014best,wilson2017good} \index{scientific computing} Scientific computing is a set of practices developed to help you ensure that the computer is being used to improve your efficiency, so that you can focus on the real-world problems instead of technical ones.\sidenote{ \url{https://www.dropbox.com/s/wqefknwfb91kop8/Coding_For_Econs_20190221.pdf?raw=1}} This means getting to know your computer a little better than most people do, and thinking critically about tasks like file structures, code and \textbf{process reusability},\sidenote{ \url{http://blogs.worldbank.org/opendata/making-analytics-reusable}} and software choice. Most importantly, it means detecting early warning signs of \textbf{process bloat}. As a general rule, if the work required to maintain a process grows as fast (or faster) than the number of objects controlled by that process, you need to stop work immediately and rethink processes. You should work to design processes that are close to infinitely scalable by the number of objects being handled -- whether they be field samples, data files, surveys, or other real or digital objects.  The first thing you need to figure to use your computer efficiently is where you are on your file system. \marginnote{You should \textit{always} use forward slashes (\texttt{/}) in file paths. Backslashes will break folder paths in many systems.}
-
-Find your \textbf{home folder}. On MacOS, this will be a folder with your username.
-On Windows, this will be something like ``This PC''. (It is never your desktop.)
-Nearly everything we talk about will assume you are starting from here.
+  \index{password protection}
+All machines should have \textbf{hard disk encryption} enabled.
+  \index{encryption}
+Disk encryption is built-in on most modern operating systems;
+the service is currently called BitLocker on Windows or FileVault on MacOS.
+Disk encryption prevents your files from ever being accessed
+without first entering the system password.
+This is different from file-level encryption,
+which makes individual files unreadable without a specific key.
+(We will address that in more detail later.)
+As with all critical passwords, your system password should be strong,
+memorable, and backed up in a separate secure location.
+
+Make sure your computer is backed up to prevent information loss.
+  \index{backup}
+Follow the \textbf{3-2-1 rule}: maintain 3 copies of all critical data,
```

Comment:
I am not sure everyone reading this will interpret _critical data_ the way it is intended there. In https://github.com/worldbank/d4di/issues/183 we talk about using the term master data. 

The main point is that we should back the rawest version of all data? All other data can be derived from that. that is the point right? So what about this:

_ maintain 3 copies of the original version of all data_

Reply:
6510324e00564293b22c9292f3a377cc62531590